src/JBController.sol

```diff
@@ -644,9 +644,9 @@ contract JBController is JBPermissioned, ERC2771Context, ERC165, IJBController,
             permissionId: JBPermissionIds.ISSUE_TOKEN
         });
 
-        bytes32 _salt = salt == bytes32(0) ? salt : keccak256(abi.encodePacked(_msgSender(), salt));
+        salt = salt == bytes32(0) ? salt : keccak256(abi.encodePacked(_msgSender(), salt));
```

Comment:
@simplemachine92

Reply:
@xBA5ED do you think it's useful to emit the original salt in an event so that it's easy to find for future deploys who want to reuse it? or is this easy to track by virtue that it was a param to the fn?